src/main/java/project/controllers/AddDriverToCarController.java

```diff
@@ -0,0 +1,41 @@
+package project.controllers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import project.lib.Injector;
+import project.models.Car;
+import project.models.Driver;
+import project.service.CarService;
+import project.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("project");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/addDriverToCar.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        Long driverId = Long.valueOf(request.getParameter("driverId"));
+        Long carId = Long.valueOf(request.getParameter("carId"));
```

Comment:
it would be nice to check if input data valid(for example empty input or letters for id)

Reply:
@YuraBondarenko this should be done in JSPs